widgets/HTTP-batchsource.json

```diff
@@ -101,6 +102,60 @@
         }
       ]
     },
+    {
+      "label": "OAuth2",
+      "properties": [
+        {
+          "widget-type": "radio-group",
```

Comment:
existing, but for new plugins I think we should use the toggle widget for booleans.

Reply:
done